PP-495: pbsnodes -l output trims the hostname to 20 characters

rampranesh opened the pull request:
<!--- Please review your changes in preview mode -->
<!--- Provide a general summary of your changes in the Title above -->

#### Issue-ID
<!--- Replace XXXX below (2 places) with the actual JIRA id -->
* **[PP-495](https://pbspro.atlassian.net/browse/PP-495)**

#### Problem description
* pbsnodes -l output trims the hostname to 20 characters

#### Cause / Analysis
* Currently the hostname is resticted to 20 bytes while printing

#### Solution description
* We are removing the length restrictions of the output

#### Checklist:
<!--- Use the preview button to see the checkboxes/links properly. -->
<!--- Go over the following points, and put an `x` (without spaces around it) in the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [x] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [x] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [x] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [x] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [x] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [x] I have attached **test logs to the Jira ticket** as evidence of testing/verification.


__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/PBSPro/pbspro?pullRequest=187) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/PBSPro/pbspro?pullRequest=187) before we can accept your contribution.<br/>

vinodchitrali commented:
Code looks Fine
Sign off

arungrover commented on test/tests/functional/pbs_pbsnodes_output_trimmed.py:
```diff
@@ -0,0 +1,93 @@
+            '12345678901234567890',
+            '123456789012345']
+        tmpip = ['127.0.0.1', '127.0.0.2', '127.0.0.3']
+        hosts = open("/etc/hosts", "a")
```
@rampranesh I don't think we should be modifying /etc/hosts on a machine. If someone cancels the test while it is running, it will just leave this modified hosts file on the system.
I think you can do one of the following:
- Instead of automating it, just add manual steps in the ticket and that to me is fine given that you need a hostname bigger than 20 chars
- If you decide to go with ptl then expect the mom's hostname to be more than 20 chars, if not then just skip the test.
- I was also thinking to use HOSTALIASES env variable but I don't think that it will work because the server would already be running by the time you will set this env variable

arungrover commented:
Apart from that PTL change, code looks good to me.

rampranesh commented:
Hi @arungrover,
        Thanks for your comments, i have removed the PTL test case and added the manual test case and logs to the ticket. Please have a look at the changes.

Thanks,
Rampranesh M

arungrover commented:
@rampranesh change looks good to me. Since it has change since the time it was reviewed by @vinodchitrali , you need another reviewer to review it.

suresh-thelkar commented:
Code changes look good to me. I sign off.

rampranesh commented:
Hi @arungrover ,
        I have got two sign-off and i have re-based my branch to the latest. Could you please take this 
PR further.

Thanks,
Rampranesh M

arungrover commented:
All looks good. Merging it in
